NOTICE

```diff
@@ -3,10 +3,3 @@ Copyright 2011-2016 The Apache Software Foundation
 
 This product includes software developed at
 The Apache Software Foundation (http://www.apache.org/).
-
-This product includes the following work from the Apache Thrift project:
-
-TNonblockingServerSocket.java
```

Comment:
We bundle it (and other ASF jars we don't mention). LEGAL-234 says this isn't needed, because the applicable copyright notice for the ASF already covers it.